P/Pfapack/build_tarballs.jl

```diff
@@ -0,0 +1,62 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Pfapack"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/xrq-phys/Pfapack.git", "0c71536b9641a0c8f4da67b373e3d4d5514561ab")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd Pfapack
+
+cmake fortran \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release
+make -j${nproc} VERBOSE=1
+make install
+
+# Manual conversion from static to dynamic lib.
+# Avoid linking libgfortran (these .f files make no reference to Fortran libs.)
+cc -shared -Wl,-force_load,${prefix}/lib/libpfapack.a \
+    -o ${prefix}/lib/libpfapack.${dlext} \
+    -L${libdir} -lblastrampoline -lm \
+
+# Copy license file
+install_license LapackLicence
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc="musl"),
+    Platform("armv7l", "linux"; libc="glibc"),
+    Platform("x86_64", "windows"),
+    Platform("x86_64", "macos"),
+    Platform("x86_64", "linux"; libc="glibc"),
+    Platform("aarch64", "linux"; libc="glibc"),
+    Platform("aarch64", "macos"),
+    Platform("x86_64", "freebsd")
+]
+platforms = expand_cxxstring_abis(platforms)
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libpfapack", :pfapack)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="libblastrampoline_jll", uuid="8e850b90-86db-534c-a0d3-1478176c7d93")),
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
```

Comment:
Whys is this necessary?
```console
% readelf -d  libpfapack.so | grep NEEDED
 0x0000000000000001 (NEEDED)             Shared library: [libblastrampoline.so.5]
 0x0000000000000001 (NEEDED)             Shared library: [libm.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
```
I don't see any library from `CompilerSupportLibraries_jll`.